packages/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -90,25 +90,11 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
     return value;
   }
 
-  /// hive does not support directly storing emoji characters. So keys are encoded in [HiveKeyStoreHelper.prepareKey] using utf7 before storing.
+  /// hive does not support directly storing emoji characters, therefore keys
+  /// are encoded in [HiveKeyStoreHelper.prepareKey] using utf7 before storing.
   @override
   Future<dynamic> put(String key, AtData? value,
-      {int? time_to_live,
-      int? time_to_born,
-      int? time_to_refresh,
-      bool? isCascade,
-      bool? isBinary,
-      bool? isEncrypted,
-      String? dataSignature,
-      String? sharedKeyEncrypted,
-      String? publicKeyChecksum,
-      String? encoding,
-      String? encKeyName,
-      String? encAlgo,
-      String? ivNonce,
-      String? skeEncKeyName,
-      String? skeEncAlgo,
-      bool skipCommit = false}) async {
+      {bool skipCommit = false}) async {
```

Comment:
The major refactoring change is here - eliminate all of these optional parameters as they were basically never used any more by the atServer code, and were being handled inconsistently if they were.